docs/network protocol.md

```diff
@@ -31,6 +31,9 @@ There are also a number of community-supported libraries available that implemen
 | GameMaker: Studio 2.x+        | [see Discord](https://discord.com/channels/731205301247803413/1166418532519653396)                 |                                                                                 |
 
 ## Synchronizing Items
+After a client connects, it will receive all previously collected items for its associated slot in a [ReceivedItems](#ReceivedItems) packet. This will include items the client may already have processed in a previous play session.
+To make it possible to reject those items if the client shouldn't process them again, each item in the packet will have an associated `index` argument. You will need to find a way to save the "last processed item index" to the player's local savegame, a local file, or something to that effect. Before connecting, you should load that "last processed item index" value and compare against it in your received items handling.
```

Comment:
To make it possible to reject those items, and if the client shouldn't process them again.

Reply:
that sounds better when I said it aloud and I mean I automatically said that anyway